client/scss/components/_footer.scss

```
@@ -85,12 +85,31 @@
   width: 100%;
 
   @include media-breakpoint-up(sm) {
-    width: 250px;
+    width: theme('spacing.64');
   }
 
   &--primary {
+    // Support basic layout support for items added via `extra_footer_actions`
+
+    align-items: stretch;
+    display: grid;
+    gap: theme('spacing.2');
+    grid-auto-flow: column;
+
+    > * {
```

From what I've seen we're only doing this to reset the default `.button` styles, so I think it's better to apply this more conservatively, e.g.

```suggestion
    > .button {
```